README.rst

```diff
@@ -34,3 +34,16 @@ Pandas on Ray
 +----------------------------------------------+-------------------------------------------------+
 
 **Pandas on Ray is currently for experimental use only. Requests and contributions are welcome!**
+
+More information and Getting Involved
+-------------------------------------
+
+- `Documentation`_
+- Ask questions on our mailing list `modin-dev@googlegroups.com`_.
+- Please report bugs by submitting a `GitHub issue`_.
```

Comment:
Please submit bug reports to our `GitHub Issues page`_.

You can also remove the please.